server/fishtest/views.py

```diff
@@ -1188,6 +1188,16 @@ def tasks(request):
         "show_task": show_task,
     }
 
+@view_config(route_name="machines", renderer="machines.mak")
```

Comment:
DEV updated with this change:
```diff
diff --git a/server/fishtest/views.py b/server/fishtest/views.py
index c21b302..37edba3 100644
--- a/server/fishtest/views.py
+++ b/server/fishtest/views.py
@@ -1189,7 +1189,7 @@ def tasks(request):
         "show_task": show_task,
     }

-@view_config(route_name="machines", renderer="machines.mak")
+@view_config(route_name="tests_machines", renderer="machines.mak")
 def machines(request):
     machines_list = request.rundb.get_machines_cache()
     if machines_list is None:
```

Reply:
> DEV updated with this change:
> 
> ```diff
> diff --git a/server/fishtest/views.py b/server/fishtest/views.py
> index c21b302..cce38ba 100644
> --- a/server/fishtest/views.py
> +++ b/server/fishtest/views.py
> @@ -1189,8 +1189,8 @@ def tasks(request):
>          "show_task": show_task,
>      }
> 
> -@view_config(route_name="machines", renderer="machines.mak")
> -def machines(request):
> +@view_config(route_name="tests_machines", renderer="machines.mak")
> +def tests_machines(request):
>      machines_list = request.rundb.get_machines_cache()
>      if machines_list is None:
>         raise exception_response(404)
> ```

DEV doesn't seem to be updated with the PR

Reply:
It didn't worked, no errors in the logs, so I started testing other PRs.